lbz/router.py

```diff
@@ -1,17 +1,14 @@
-"""
-Router module.
-"""
+"""Router module."""
```

Comment:
I would consider removing that and Router class below.

Reply:
I thought about removing docstrings in many places as they were soooo obvious, will remove them 😌